data/io.elementary.terminal.gschema.xml

```diff
@@ -173,5 +178,10 @@
       <summary>Alert the user about unsafe paste.</summary>
       <description>If the user paste's a command into the terminal that might be considered unsafe, warn them.</description>
     </key>
+    <key name="sync-zoom" type="b">
+      <default>false</default>
+      <summary>Whether all tabs have the same zoom</summary>
+      <description>Whether changes to one tab's zoom level should be applied to all tabs</description>
+    </key>
```

Comment:
I don't think we should add this to be honest. It's such a specific, finicky setting. We should just ensure the default behavior makes sense. :smile: 

```suggestion
```